sample-exam-answers.tex

```diff
@@ -0,0 +1,34 @@
+\documentclass{istqb}
```

Comment:
The CI assumes that every `*.tex` file has a directory `*/`. This allows us to skip producing an EPUB/HTML version for file `*.tex` if `*/NO_HTML` exists. Therefore, we should add symlinks `sample-exam-questions` and `sample-exam-answers` that point to the `sample-exam/` directory.